yo1k/tic_tac_toe/core/game.py

```diff
@@ -35,45 +35,47 @@ class Cell:
     def __init__(self, x: int, y: int):
         assert 0 <= x < Board.size(), f"{x}, {Board.size()}"
         assert 0 <= y < Board.size(), f"{y}, {Board.size()}"
-        self.x = x
-        self.y = y
+        self.x: int = x
+        self.y: int = y
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return (f"{type(self).__qualname__}("
                 f"x={self.x},"
                 f"y={self.y})")
 
 
 @eq
 class Board:
-    def __init__(self, cells: Optional[Sequence[list[Optional[Mark]]]] = None):
-        self.cells = Board.__empty_cells() if cells is None else cells
+    def __init__(self, cells: Optional[MutableSequence[MutableSequence[Optional[Mark]]]] = None):
+        self.cells: MutableSequence[MutableSequence[Optional[Mark]]] \
+            = Board.__empty_cells() if cells is None else cells
         Board.__assert_board(self.cells, Board.size())
 
-    def set(self, cell: Cell, mark: Mark):
+    def set(self, cell: Cell, mark: Mark) -> None:
         self.cells[cell.x][cell.y] = mark
 
     def get(self, cell: Cell) -> Optional[Mark]:
         return self.cells[cell.x][cell.y]
 
-    def clean(self):
+    def clean(self) -> None:
```

Comment:
Let's rename `clean` -> `clear`, to be consistent with [`MutableSequence.clear`](https://docs.python.org/3/library/stdtypes.html#mutable-sequence-types)

Reply:
Done.